test/integration/narrow/routes/cdo/create/owner-details.test.js

```diff
@@ -216,6 +216,43 @@ describe('OwnerDetails test', () => {
     })
   })
 
+  test('POST /cdo/create/owner-details updates owner details session correctly given one exists and DOB is empty', async () => {
```

Comment:
It might be worth adding a test for when there is a partial DOB when the 'delete ownerDetails/dateOfBirth' gets run